src/components/InputBook.css

```diff
@@ -0,0 +1,62 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800;900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@400;500;600;700;800&display=swap");
+
+form {
+  padding-top: 1.813rem;
+  padding-bottom: 15.563rem;
+}
+
+form > h2 {
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  letter-spacing: -0.18px;
+  color: #888;
+  margin-bottom: 1.188rem;
+}
+
+.flex-grp {
```

Comment:
- [ ] Please the styling is exactly the same as the one provided on the requirement. ([ see the big picture of this project](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md).)
- [ ] - Your design. 
![cd](https://user-images.githubusercontent.com/63972670/167233796-5b57b21a-7e25-4297-b321-ab76e79468aa.JPG)

Reply:
@OLIPLICHE thank you for the changes you requested, I have corrected them